src/pocketmine/player/Player.php

```diff
@@ -2147,6 +2150,34 @@ public function onFormSubmit(int $formId, $responseData) : bool{
 		return true;
 	}
 
+	/**
+	 * @internal
+	 * @param Scoreboard $scoreboard
+	 */
+	public function addScoreboard(Scoreboard $scoreboard) : void{
```

Comment:
Correct me if i'm wrong, but a player can only see one scoreboard at a time? It does not make sense to give the player multiple scoreboards if it can only see one of them.

I mean it may be possible to have one on the sidebar and lets say below their name, but afaik Bukkit hasn't done this so there may be a reason for that?

Reply:
@lukeeey Player can see sidebar and list at the same time